tests/python/shared/fixtures/data.py

```diff
@@ -60,6 +62,50 @@ def tasks():
         return Container(json.load(f)["results"])
 
 
+def filter_assets(resources: Iterable, **kwargs):
+    filtered_resources = []
+    exclude_prefix = "exclude_"
+
+    for resource in resources:
+        for key, value in kwargs.items():
+            if exclude := key.startswith(exclude_prefix):
+                key = key[len(exclude_prefix) :]
+
+            op = operator.ne if exclude else operator.eq
```

Comment:
This is a confusing use of the `:=` operator, because the variable is not used in the `if` body (but is used outside of it). I would suggest this instead:

```suggestion
            op = operator.eq
            
            if key.startswith(exclude_prefix):
                key = key[len(exclude_prefix) :]
                op = operator.ne
```